### PR TITLE
Sawn Off Weapons

### PR DESCRIPTION
**Pull request by ChronicPwnage**
## Description

Adds some sawn-off variants of a few weapons, as well as makes sawing off a gun more impactful and interesting. Sawn-off guns are less accurate, have more recoil, don't hit as hard as a blunt weapon, and sawn-off shotguns have greater pellet dispersion.

## Motivation and Context

It's a cool little addition and makes it so that people have more options for the weapons they carry.
## How Has This Been Tested?

Tested on private server, no issues so far.
## Changelog (neccesary)
:cl:
add: Adds sawn-off variants to the remington, scoped remington, caravan shotgun, and single shotgun.
tweak: Sawn-off guns are now less accurate, have more recoil, and (shotguns) have greater pellet dispersion
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -262,14 +263,15 @@
 // RIOT SHOTGUN //
 /obj/item/gun/ballistic/shotgun/riot //for spawn in the armory
 	name = "light shotgun"
-	desc = "A light weight shotgun chambered in 12 gauge, designed to be used by paramilitary companies during air raids, but was later absorbed by Vault-tec for use by security in their vaults."
+	desc = "A lightweight shotgun chambered in 12 gauge, designed to be used by paramilitary companies during air raids, but was later absorbed by Vault-tec for use by security in their vaults."
 	icon_state = "riotshotgun"
 	item_state = "shotgunriot"
-	force = 40
+	force = 25
```
Comment (Light-Weave) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]: stealth nerf to smashing people in the face with heavy weapons?

Comment (ChronicPwnage) [reply code/modules/projectiles/guns/ballistic/shotgun.dm]: "Light" shotgun
40 force damage

I don't think it should be a robust melee weapon on top of the highest capacity shotgun that's commonly found.

Comment (Light-Weave) [reply code/modules/projectiles/guns/ballistic/shotgun.dm]: yeah originally it had a desc of like "this is a shotgun with a super bulky reinforced stock of cracking heads" but someone changed it

Comment (GhostEcho) [reply code/modules/projectiles/guns/ballistic/shotgun.dm]: Designed to be a buff melee weapon, and was autistically renamed by oxford, please change back, keeping in mind most armors will bring it down to about 20/25

**Pull request merged**